e2e/tools/validator/query.json

```diff
@@ -1,18 +1,4 @@
 [
-    {
-        "expected_query": "rate(kepler_{level}_package_joules_total{{{query}, mode='dynamic'}}[{interval}])",
-        "actual_query": "rate(kepler_node_platform_joules_total[{interval}])"
-    },
-    {
-        "expected_query": "rate(kepler_{level}_platform_joules_total{{{query}, mode='dynamic'}}[{interval}])",
-        "actual_query": "rate(kepler_node_platform_joules_total[{interval}])"
-    },
-    {
-        "expected_query": "rate(kepler_{level}_bpf_cpu_time_ms_total{{{query}}}[{interval}])",
-        "actual_query": "sum by(__name__, job) (rate(kepler_process_bpf_cpu_time_ms_total[{interval}]))"
-    },
-    {
-        "expected_query": "rate(kepler_{level}_bpf_page_cache_hit_total{{{query}}}[{interval}])",
-        "actual_query": "sum by(__name__, job) (rate(kepler_process_bpf_page_cache_hit_total[{interval}]))"
-    }
+    "abs(1 - (rate(kepler_{level}_package_joules_total{{{query}, mode='dynamic'}}[{interval}]) / rate(kepler_node_platform_joules_total{{job='vm'}}[{interval}])))",
```

Comment:
can you make `job='vm'` a param, so i can replace the vm name?

Reply:
How about we require a label to indicate `vm` other than `job` ? 
e.g. `target="vm"` ? and `target="metal"` ?

Reply:
Converted job='vm' into a param. For now I put job='vm' as the placeholder